### ISSUE TITLE
runtime error

### ISSUE THREAD

**Issue by Satiel** [chenasraf/treelike#2]
## Expected Behavior

Expected a treelike result

## Current Behavior

I get the following error: 
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x10 pc=0x4a43af]

goroutine 1 [running]:
main.parseInput(0xc00000c2c0, 0x2f, 0xc000042040)
        /github/workspace/treelike.go:129 +0x1ff
main.main()
        /github/workspace/treelike.go:292 +0x372

```
## Possible Solution

N/A

## Steps to Reproduce

1. Created a text file in notepad, named it 'test.txt'
2. Put the following into the text file (indentations made with spaces):
```
I
 am
  a
   superhero!
a
 what?
a
 superhero!
```

3.  Run treelike with the following parameters from a CDM prompt in the same directory as treelike and the text file: 
`treelike -f test.txt`
4.
5.

## Screenshot
![treelike_runtime_error](https://github.com/user-attachments/assets/fd286268-ccef-4e47-9a6f-f839bfd8ec98)

**Comment by Satiel** [comment]
Here is the text file that was used to reproduce that error
[test.txt](https://github.com/user-attachments/files/16830616/test.txt)

**Comment by chenasraf** [comment]
@Satiel thanks for the report! :) I'll get on it

**Comment by chenasraf** [comment]
Should be fixed in #3, let me know if it still gives you problems
